packages/insomnia-app/app/ui/components/wrapper-design.js

```diff
@@ -235,7 +247,29 @@ class WrapperDesign extends React.PureComponent<Props, State> {
               'preview-hidden': previewHidden,
             })}>
             <div id="swagger-ui-wrapper">
-              <SwaggerUI spec={swaggerUiSpec || {}} supportedSubmitMethods={['get', 'put', 'post', 'delete', 'options', 'head', 'patch', 'trace']} />
+              <ErrorBoundary
+                invalidateWith={swaggerUiSpec}
+                replaceWith={
+                  <div className="text-center margin">
+                    <h3>An error occurred while trying to render Swagger UI 😢</h3>
+                    This preview will automatically refresh, once you have a valid specification
+                    that can be previewed. If it doesn't, try re-opening the document.
+                  </div>
+                }>
+                <SwaggerUI
+                  spec={swaggerUiSpec || {}}
```

Comment:
Is parsing the apiSpec necessary? We _can_ send the original contents (`activeApiSpec.contents`) directly to SwaggerUI, and it handles the error case fixed by this PR correctly, however we have no control over the error it shows.

With the following code change, SwaggerUI shows this in the preview
```suggestion
                  spec={activeApiSpec.contents}
```
![image](https://user-images.githubusercontent.com/4312346/81244695-35640500-9067-11ea-9859-777e3e7a9198.png)

Reply:
If it works, I'm fine with passing it straight through